backend/src/lib/controllers/dashboard-ctrl.ts

```diff
@@ -224,23 +224,31 @@ async function publishDashboard(req: Request, res: Response) {
 async function publishPendingDashboard(req: Request, res: Response) {
   const user = req.user;
   const { id } = req.params;
-  const { updatedAt } = req.body;
+  const { updatedAt, releaseNotes } = req.body;
 
   if (!updatedAt) {
     res.status(400).send("Missing required body `updatedAt`");
     return;
   }
 
   const repo = DashboardRepository.getInstance();
-
   const dashboard = await repo.getDashboardById(id);
-  if (dashboard.state !== DashboardState.Draft) {
+  if (
+    dashboard.state !== DashboardState.Draft &&
+    dashboard.state !== DashboardState.PublishPending
+  ) {
     res.status(409);
     return res.send("Dashboard must be in draft state");
```

Comment:
Can you update this message due to now we also accept PublishPending dashboard state?

Reply:
Good catch. Will do